sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -796,3 +797,63 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function which returns all the keys of outer JSON object.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(json_object) - returns all the keys of outer JSON object.",
+  arguments = """
+    Arguments:
+      * json_object - A JSON object is required as argument. `Null` is returned, if an invalid JSON
+          string is given. `Runtime Exception` is thrown, if null string or JSON array is given.
```

Comment:
How about this?
```
      * json_object - A JSON object. If it is an invalid string, the function returns null.
              If it is a JSON array or null, a runtime exception will be thrown.
```
btw, the error handling is consistent with the other JSON functions?

Reply:
I checked the other json functions. `IllegalArgumentException` is thrown by `from_json` for invalid inputs and it is better than `RuntimeException`.